app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -102,7 +102,7 @@ window.GOVUK.analyticsGA4.analyticsModules = window.GOVUK.analyticsGA4.analytics
     // return only the date from given timestamps of the form
     // 2022-03-28T19:11:00.000+00:00
     stripTimeFrom: function (value) {
-      if (value !== null) {
+      if (value !== undefined) {
```

Comment:
Is this a related change? I'm just interested in how this works... did something fail if this was left as `null`?

Reply:
@andysellick yep, without changing this, the if statement will return true as `undefined !== null`, but then it will crash as it tries to do `undefined.split()`

Reply:
Oh of course if there isn't a value it will default to being the undefined default... so it is a related change. Thanks.

Reply:
Maybe it should be `value !== null && value != undefined` ?